### DISCUSSION TITLE
Allow all domains in dangerousRemoteDomainIpcAccess

### DISCUSSION THREAD

**Discussion by binbard** [tauri-apps/tauri#11153] [Q&A]
I don't care about security, I just want every remote url can execute the tauri apis

"security": {
    "dangerousRemoteDomainIpcAccess": [
        { "domain": "tauri.app", "windows": ["tauri"], "apiMethods": ["event"] }
    ]
}

**Comment by FabianLars** [comment]
This is not supported, and likely won't be supported. You can however extend it in rust at runtime: https://docs.rs/tauri/1.8.1/tauri/scope/ipc/struct.Scope.html (v1) / https://docs.rs/tauri/2.0.0-beta/tauri/trait.Manager.html#method.add_capability (i think that's the v2 alternative)
